infrastructure/parameters.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Overall suggestion: use the Azure naming conventions in naming:
https://docs.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/resource-naming
https://docs.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/resource-abbreviations

Reply:
will study it, too lazy to fix now 🥇